packages/components/src/styles/components/form/super-select.scss

```diff
@@ -0,0 +1,487 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// SUPER-SELECT
+
+// Copied from components/src/styles/components/tag.scss styles
+// we set a higher value than the line-height (~13px) to accommodate cases where the text wraps
+$hds-tag-border-radius: 50px; // TODO: This variable is repeated from tag.scss
+$hds-super-select-item-height: 36px;
+
+// Style overrides:
+
+.hds-form-super-select {
+  --hds-form-super-select-dropdown-max-width: none;
+
+  width: 100%;
+}
+
+.hds-form-super-select .ember-basic-dropdown {
+
+  // TRIGGER
+  .ember-basic-dropdown-trigger,
+  .ember-power-select-trigger {
+    display: flex;
+    align-items: center;
+    max-width: 100%;
+    min-height: $hds-super-select-item-height;
+    padding: 4px;
+    padding-right: calc(var(--token-form-control-padding) + 24px); // extra space for the icon
+    padding-left: 11px;
+    color: var(--token-form-control-base-foreground-value-color);
+    line-height: var(--token-typography-body-200-line-height);
```

Comment:
This is required to override the PowerSelect styles that set it at `1.75` (we want it at `1.4286` to be the same with Select and other components, otherwise the total height will be 38px instead of 36px)